study.md

```diff
@@ -151,7 +155,13 @@ What are the 4 most common HTTP verbs used when creating a RESTful API. How
 would you use each?
 
 ```md
-<!-- your answer here -->
+CREATE: The create or add new entries
```

Comment:
This is what they do, but I think they are looking for the verbs themselves: GET, POST, PATCH, DELETE